sandbox/rocky/tf/algos/ddpg/ddpg_base_net.py

```diff
@@ -0,0 +1,220 @@
+import tensorflow as tf
+import tensorflow.contrib as tc
+import numpy as np
+
+
+def _fc(x,
+        output_dim,
+        weight_initializer=tf.contrib.layers.xavier_initializer(uniform=False),
+        bias_initializer=tf.zeros_initializer):
+    return tf.contrib.layers.fully_connected(x, output_dim, activation_fn=None, \
+            weights_initializer=weight_initializer, weights_regularizer=None, biases_initializer=bias_initializer)
+
+
+def _norm(x):
+    return tc.layers.layer_norm(x, center=True, scale=True)
+
+
+class Model(object):
```

Comment:
For actor critic, I think we could work on q functions and policies that is implemented directly so they are reusable.

Reply:
I could not find the q functions and policies functions in tensorflow. If there are, could you provide them for me?